wen-restart/src/wen_restart.rs

```diff
@@ -157,17 +158,14 @@ pub(crate) fn aggregate_restart_last_voted_fork_slots(
         let result = last_voted_fork_slots_aggregate.get_aggregate_result();
         let mut filtered_slots: Vec<Slot>;
         {
-            let my_bank_forks = bank_forks.read().unwrap();
             filtered_slots = result
                 .slots_to_repair
                 .into_iter()
                 .filter(|slot| {
                     if slot <= &root_slot || is_full_slots.contains(slot) {
                         return false;
                     }
-                    let is_full = my_bank_forks
-                        .get(*slot)
-                        .map_or(false, |bank| bank.is_frozen());
+                    let is_full = blockstore.is_full(*slot);
                     if is_full {
```

Comment:
think we can just move this to one line `if blockstore.is_full(*slot) {}`

Reply:
Done.